src/pages/ContactList/ContactList.jsx

```diff
@@ -0,0 +1,17 @@
+import "./ContactList.scss";
+import ContactContent from "../../components/ContactContent/ContactContent";
+// import { getUserContacts } from "../../utils/apiUtils";
+// import { useState } from "react";
+
+const ContactList = () => {
+
+// const [userData, setUserData] = UseState([]);
+
+    return (
+        <div className="contactList">
+            <ContactContent className="contactList__main" />
```

Comment:
do not add a `className` like this, it will be read as a prop, also it's unnecessary